src/content/smart-contracts/index.md

```diff
@@ -29,47 +23,43 @@ Alice and Bob are having a bicycle race. Let's say Alice bets Bob $10 that she w
 
 This silly example illustrates the problem with any non-smart agreement. Even if the conditions of the agreement get met (i.e. you are the winner of the race), you must still trust another person to fulfill the agreement (i.e. payout on the bet).
 
-## Smart contracts {#smart-contracts}
-
-Smart contracts digitize agreements by turning the terms of an agreement into computer code that automatically executes when the contract terms are met.
-
 ### A digital vending machine {#vending-machine}
 
 A simple metaphor for a smart contract is a vending machine, which works somewhat similarly to a smart contract - specific inputs guarantee predetermined outputs.
 
 - You select a product
-- The vending machine returns the amount required to purchase the product
-- You insert the correct amount
-- The vending machine verifies you have inserted the correct amount
-- The vending machine dispenses the product of choice
+- The vending machine displays the price
+- You pay the price
+- The vending machine verifies that you paid the right amount
+- The vending machine gives you your item.
 
 The vending machine will only dispense your desired product after all requirements are met. If you don't select a product or insert enough money, the vending machine won't give out your product.
 
 ### Automatic execution {#automation}
 
-One of the most significant benefits smart contracts have over regular contracts is that the outcome is automatically executed when the contract conditions are realized. There is no need to wait for a human to execute the result. In other words: smart contracts remove the need for trust.
+The main virtue of a smart contract is that it automatically executes the outcome when its conditions are met. There is no need to wait for a human to execute the result. This removes the need for a  trusted intermediary.
```

Comment:
1. smart contracts don't remove the need for *trust* en toto, i.e. you still need to trust the Ethereum network's stability and security 😃 . How about "This removes the need for interpersonal trust?"
2. virtue has a secondary meaning of "a good or useful quality of a thing, e.g. "Mike was extolling the virtues of the car," but it would be totally fine to replace it with 'advantage' or 'benefit'

Reply:
1. I don't really disagree but doesn't this concept go against the idea of "Don't trust, Verify" 🤔. I suppose the idea is that you can verify the code instead of trusting a black box.
2. Yeah, I get that but we need to remember that we're serving a lot of non-native English speakers, so we prefer to tend towards easier-to-understand over purple prose.

Reply:
Agree that none of this is completely trust free; the key point is of course not needing to trust another human, and instead you must trust the code instead, which is open instead of in a black-box. 

Somewhat indifferent towards "virtue" but would probably err on the side of using a more direct terms here, similar to what it was.

Reply:
```suggestion
The main benefit of a smart contract is that it deterministically executes unambiguous code when certain conditions are met. There is no need to wait for a humans to interpret or negotiate the result. This removes the need for trusted intermediaries.
```

Reply:
what does the phrase 'ambiguous code' mean? Ambiguous as in, a human reader might interpret it one of a few ways? Because you could also say that the whole smart contract idea hinges on the code _not_ being ambiguous, at least, not to the computer. Otherwise the contract couldn't deterministically execute.

Reply:
it says "unambiguous".